doc/api/addons.md

```diff
@@ -1096,8 +1096,10 @@ has ended but before the JavaScript VM is terminated and Node.js shuts down.
 
 #### void AtExit(callback, args)
 
-* `callback` {void (\*)(void\*)} A pointer to the function to call at exit.
-* `args` {void\*} A pointer to pass to the callback at exit.
+* `callback` <span class="type">&lt;void (\*)(void\*)&gt;</span>
+  A pointer to the function to call at exit.
+* `args` <span class="type">&lt;void\*&gt;</span>
```

Comment:
were these changed so they don't trigger the type parser?

Reply:
Currently, `tools/doc/html.js` type-process any non-code text in any section, so it cannot define if this is a js type or something else. But the author of this section wanted the C types to be consistent in CSS style with other parts, so I've tried to conform.